src/wlapplication.cc

```diff
@@ -825,6 +825,7 @@ bool WLApplication::init_settings() {
 	get_config_bool("single_watchwin", false);
 	get_config_bool("ctrl_zoom", false);
 	get_config_bool("game_clock", true);
+	get_config_bool("numpad_diagonalscrolling", false);
```

Comment:
Diagonal scrolling should be disable by default, because it conflicts with time control.